harpy/bin/deconvolve_alignments.py

```
@@ -0,0 +1,204 @@
+        LAST_CONTIG = chrm
+        continue
+
+    try:
+        bx = record.get_tag("BX")
+        # do a regex search to find X00 pattern in the BX
+        if re.search("[ABCD]0{2,4}", bx):
+            # if found, invalid
+            write_invalidbx(outfile, record)
+            LAST_CONTIG = chrm
+            continue
+    except:
```

Ruff

Do not use bare `except`

(E722)